### PR TITLE
19 add custom ggplot theme

### LINKED ISSUE #19: include a 2dii custom ggplot theme
Include a 2dii custom ggplot theme that could combine official 2dii color palette/s, official 2dii font specs, and other data viz guidances from @MonikaFu 

Such a custom ggplot theme could then be applied to any custom ggplot that a 2dii user may be creating.

see this package for some cool, complex examples: https://github.com/hrbrmstr/hrbrthemes

I believe @vintented may have made something like this before, or was that an Rmarkdown theme?

### PR DESCRIPTION
**Pull request by MonikaFu**
Closes #19 

In this PR I add a function returning a ggplot theme with 2DII aesthetics and use it in the rest of `r2dii.ggplot` plotting functions. 

In line 276 there is a secondary change to improve the aesthetics of the tech mix chart for power. Commit 30a53bf

Testing:
Run the example file with updated functions and the charts are successfully produced with the right aesthetics.

### CODE REVIEW

**Review by cjyetman — APPROVED** [review]
this is rad! tested on my machine and works as expected. I have a follow-on suggestion that I will open a new issue for. this PR lgtm

File: R/plot_2diicharts.R [hunk 1]
```diff
@@ -1,37 +1,43 @@
...
-      margin = margin(5, 5, 5, 5)
-    )) +
-    theme(axis.title = element_text(
-      family = font_family,
-      size = font_size_axis_titles,
-      margin = margin(5, 5, 5, 5)
-    ))
-
-  return(p_general)
+  theme_classic() %+replace%
```
Comment (jdhoffa) [inline R/plot_2diicharts.R:16]: hell ya `%+replace%` !! :-D

**Review by jdhoffa — APPROVED** [review]
I concur with @cjyetman on the radness enclosed in this PR. LGTM

**Comment by MonikaFu** [comment]
@jdhoffa I'm happy to see so much enthusiasm 🙂

**Pull request merged**